test/osb_and_plugin_test/osb_test/deprovision_test.go

```diff
@@ -388,4 +388,23 @@ var _ = Describe("Deprovision", func() {
 			})
 		})
 	})
+
+	Context("deprovision operation is handled by external plugin", func() {
+		BeforeEach(func() {
+			brokerServer.ServiceInstanceHandler = parameterizedHandler(http.StatusOK, `{}`)
+			ctx.SMWithBasic.PUT(smBrokerURL+"/v2/service_instances/"+SID).
+				WithHeader(brokerAPIVersionHeaderKey, brokerAPIVersionHeaderValue).
+				WithJSON(provisionRequestBodyMap()()).Expect().StatusRange(httpexpect.Status2xx)
+		})
+
+		It("osb store plugin should not create delete operation", func() {
+			shouldSaveOperationInContext = true
+			brokerServer.ServiceInstanceHandler = parameterizedHandler(http.StatusAccepted, `{"operation":"abc123"}`)
```

Comment:
this should be in BeforeEach, the It part should only do the test, not the preparations